Extensions/TweenBehavior/JsExtension.js

```diff
@@ -44,10 +44,10 @@ module.exports = {
 
     const behavior = extension
       .addBehavior(
-        "TweenBehavior",
-        _("Tween animation"),
-        "TweenBehavior",
-        _("Tween animation"),
+        "Tween",
+        _("Tween"),
+        "Tween",
```

Comment:
This one is fine: It's the name that is displayed to the user, so can be anything. "Tween" is fine.